src/services/users/types/userResponseTypes.ts

```diff
@@ -0,0 +1,50 @@
+export interface LoginResponse {
+  AccessToken: string,
+  ExpiresIn: number,
+  TokenType: string
+  IdToken: string,
+  RefreshToken: string
+}
+
+export interface SignUpResponse {
+  participant_uuid: string
+}
+
+export interface CreateUserResponse {
+  first_name: string,
+  last_name: string,
+  email: {
+    address: string,
+    verified: boolean,
+  }
+  picture: string,
+  data_created: string,
+  data_updated: string,
+  date_deleted: string,
+  last_login: string,
+  uuid: string,
+}
+
+
+export interface RefreshTokensResponse {
+  AccessToken: string,
+  ExpiresIn: number,
+  TokenType: string,
+  IdToken: string,
+}
+
+export interface ResetPasswordResponse {
+
+}
```

Comment:
Do we want an empty interface, can we just get rid of this?

Reply:
We want to have a type that corresponds to the return of every endpoint. I would be more curious whether this endpoint in users actually returns something or not. If it does, it should be reflected here, if it doesn't then users should probably be changed to return something.

Reply:
the endpoint itself it return what cognito client return. it was a big hard to find out what to return. i might need a bit help with this one

Reply:
This shouldn't be an issue when Jenn refactors users. All endpoints will need to return our own model and nothing directly from a cognito client